### DISCUSSION TITLE
Breadcrumblist

### DISCUSSION THREAD

**Discussion by ghost** [RehanSaeed/Schema.NET#37] [Q&A]
BreadcrumbList.ItemListElement should be a list, as it has to allow many itemlist elements like so:

But in the current implmentation, only one itemlistelement is allowed.

```
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [{
    "@type": "ListItem",
    "position": 1,
    "name": "Books",
    "item": "https://example.com/books"
  },{
    "@type": "ListItem",
    "position": 4,
    "name": "Ancillary Justice",
    "item": "https://example.com/books/authors/ancillaryjustice"
  }]
}
</script>
```

**Comment by RehanSaeed** [comment]
Not true. See:

https://github.com/RehanSaeed/Schema.NET/blob/13cfa2362791e076ae6af4e862aebac32af9e4e1/Tests/Schema.NET.Test/BreadcrumbListTest.cs

**Comment by benmccallum** [comment]
Hey @RehanSaeed , sorry to comment on a new issue but the following gives me compiler errors, at least in razor: 

```
new Schema.NET.BreadcrumbList()
{
    ItemListElement = new List<Schema.NET.ListItem>()
    {
        new Schema.NET.ListItem() { Name = "x", Url = new Uri("") },
        new Schema.NET.ListItem() { Name = "y", Url = new Uri("") },
    }
}
```

Cannot convert `List<ListItem>` to `Values<IListItem, string, IThing>`. Any suggestions?

**Comment by benmccallum** [comment]
Copied the unit test verbatim too and it's complaining in a .cs file as well, weird

**Comment by benmccallum** [comment]
Oh snap, figured it out. That snapshot of a unit test in time is no longer valid. Pulled latest master and it was working, as one would expect, but was using `List<IListItem>`, so the interface must be used now.
